### ISSUE TITLE
add command to repeat most recent notification 

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#177694]
With the new way of accepting a notification's primary action via keybinding, @jooyoungseo suggested it would be useful to have a command to hear the most recent notification again so one can confirm that's what they'd like to accept without losing context.

We'd have to use aria-live assertive here I think.

**Comment by Tyriar** [comment]
Can you not do this now with Show Notifications and then escape to return to the editor?

**Comment by meganrogge** [comment]
@jooyoungseo, lmk if Daniel's suggestion above does not meet your needs
